formats/proto/src/main/java/io/cloudevents/proto/JsonValueCloudEventData.java

```diff
@@ -0,0 +1,19 @@
+package io.cloudevents.proto;
+
+import io.cloudevents.CloudEventData;
+
+import javax.json.JsonValue;
+
+/**
+ * {@link CloudEventData} representing a {@link JsonValue}
+ */
+public interface JsonValueCloudEventData extends TextCloudEventData {
```

Comment:
This sounds interesting, worth merging with #348 i guess

Reply:
No wait i'm sorry, i wanted to point out to `ProtoCloudEventData`, not this one.